sklearn/tests/test_common.py

```diff
@@ -379,3 +381,37 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+
+# addresses #4467, #4465 and #4461
+# essentially tests consistency between shallow and deep
+# get_params() calls of
+# - FeatureUnion
+# - Pipeline
+def test_custom_get_params():
+    # find classes that overwrite the get_params() function
+    estimators = all_estimators(
+        include_meta_estimators=True,
+        include_other=True)
+    custom_funcs = []
+    for name, obj in estimators:
```

Comment:
Technically, it is a class, not an object, so it should probably not be called 'obj', but 'klass', ('class' cannot be used, as it is a reserved argument).